slapp/templates/pages/parameters.html

```diff
@@ -252,11 +196,21 @@ <h2>Parameter für die Energiesystemoptimieung festlegen</h2>
       <i>Je nach Anzahl der Gemeinden dauert eine Modellierung & Optimierung mehrere Minuten.</i>
       <div class="mb-3">
         <a class="btn btn-primary"
-           href="{% url 'explorer:results' %}?{% for municipality_ids in municipalities %}id={{ municipality_ids.id }}&{% endfor %}"
+           href="{% url 'explorer:results' %}?{% for municipality, mun_context in mun_forms.items %}id={{ mun_context.id }}&{% endfor %}"
```

Comment:
We can remove mun IDs here, as they are stored in session, right?

Reply:
that whole button could probably be removed since the results view is being handled by the wizard button. But I was thinking about a submit button for the forms at that spot.